### DISCUSSION TITLE
Firmware for Shelly Mini1PMG3

### DISCUSSION THREAD

**Discussion by GerdRuetten** [arendst/Tasmota#20637] [Q&A] [answered]
Dear all,

I flashed a new Shelly Mini1PMG3 with Tasmota as described here:
https://github.com/tasmota/mgos32-to-tasmota32

After the procedure the firmware is Tasmota 12.5.0.

Two questions:

a) The tutorial point 10 says "The conversion will update to the latest Tasmota version too."
But after the migration process, the tasmota version is 12.5.0 ... **Why? And why not 13.3.0??**
The OTA-URL in the process is setting to: http://ota.tasmota.com/tasmota32/release/tasmota32c3.bin

b) Is a German version of this c3-firmware also available?
At https://ota.tasmota.com/tasmota32/release/ I only see a DE version for tsmota32 but not for tasmota32c3

TIA!

Best regards
Gerd

**Comment by sfromis** [comment]
b) Due to the multiplicity of chip architectures, build variants, and languages, the language-specific precompiled builds are only available for the most common, ESP8266 and "orignal ESP32" base builds. You can do a custom build for the C3 if you like (not hard to do).

**Reply by GerdRuetten** [reply]
Thank you for your support!
I understand the limitation of one languages.
If I have time, I'd like to take a look at it with the custom build!

Best regards
Gerd

**Comment by Jason2866** [comment, marked as answer]
a) The partition is to small to flash latest Tasmota version. Before actual Tasmota can be flashed the safeboot conversion process needs to be done first. If every step was successful as described in the guide, device firmware is the actual release version. In your case a step was not done or wasn't successful. The described steps needs to be done exactly as described!

**Reply by GerdRuetten** [reply]
Thank you for your support!

Hmmm, there was no mistake and I went through the instructions step by step.

In the meantime, I also flashed a second Shelly with c3 firmware.
Here, too, the “old” firmware 12.5.0 was still on it after the first conversion.
Only when I ran the Partition Wizard a second time was it updated to the current 13.3.0.

Best regards
Gerd

**Reply by Jason2866** [reply]
> Here, too, the “old” firmware 12.5.0 was still on it after the first conversion.

Which first conversion? Step?

**Reply by GerdRuetten** [reply]
Step 10: "Start Migration" in the Partition Wizard
